docs/practices/open-sourcing.md

```diff
@@ -0,0 +1,267 @@
+# How to Open Source a Software Project
+
+This guide explains how to open source a software project. It guides you through the decisions which need to be taken and the practical steps to publish code as open source software for the first time.
+
+## Motivation
+
+There are many reasons why it can be a good idea to open source a software project:
+
+* Increasing adoption and setting standards
+* Establishing transparency and trust
+* Building partnerships and ecosystems
+* Reducing costs through shared development
+* Attracting talent and engaging employees
+
+There is excellent material available in the community which dives deep on each of these motivations and some more. Good starting points are TODO Group's ["Guide to Outbound Open Source Software"](https://todogroup.org/resources/guides/a-guide-to-outbound-open-source-software/), Bitkom's ["Open Source Guide"](https://www.bitkom.org/sites/main/files/2024-04/bitkom-opensource-guide-en.pdf) and Mozilla's ["Framework For Purposeful Open Source"](https://blog.mozilla.org/wp-content/uploads/2018/05/MZOTS_OS_Archetypes_report_ext_scr.pdf).
```

Comment:
I heard someone wrote an excellent document about indicators for good projects and brought it into OpenRail ;)
https://projects.openrailassociation.org/docs/business/indicators/ 

IMHO this should be linked at least once in this document.